java-checks/src/main/resources/org/sonar/l10n/java/rules/squid/S3750_java.html

```diff
@@ -0,0 +1,18 @@
+<p>Spring <code>@Controller</code>s, <code>@Service</code>s, and <code>@Repository</code>s have <code>singleton</code> scope by default, meaning only
+one instance of the class is ever instantiated in the application. Defining any other scope for one of these class types will result in needless churn
+as new instances are created and destroyed. In a busy web application, this could cause a significant amount of needless additional load on the
+server.</p>
+<p>This rule raises an issue when the <code>@Scope</code> annotation is applied to a <code>@Controller</code>, <code>@Service</code>, or
+<code>@Repository</code> with any value but "singleton". <code>@Scope("singleton")</code> is redundant, but ignored.</p>
+<h2>Noncompliant Code Example</h2>
+<pre>
+@Scope("prototype")  // Noncompliant
+@Controller
+public class HelloWorld {
+</pre>
+<h2>Compliant Solution</h2>
+<pre>
+@Controller
+public class HelloWorld {
```

Comment:
side remark but maybe adding an annotated compliant example could be a nice addition to documentation ? (this should be done in RSPEC)